### ISSUE TITLE
PHP open_basedir directive and password generation

### ISSUE THREAD

**Issue by evgeniypiter** [sjarvela/kloudspeaker#171]
Suggest to add note ([here](https://github.com/sjarvela/kloudspeaker/wiki/Installation) or more appropriate wiki page) about using /dev/urandom for password generation (config/users). If [open_basedir](http://php.net/manual/en/ini.core.php#ini.open-basedir) value was changed from NULL (default) to some path(s) which does not include /dev/urandom, generated password will not be saved to database.

**Comment by sjarvela** [comment]
Good point. I added mention about this at the setting (https://github.com/sjarvela/kloudspeaker/wiki/Backend-configuration-options#disable-devurandom-access-no_dev_urandom), not sure where else this could be mentioned.

Installation might not be best place, as I doubt users know about this until they run into error, perhaps troubleshooting wiki.
